packages/ui-library/src/components/textarea/index.stories.ts

```diff
@@ -319,34 +319,34 @@ export default {
     },
 
     // Events
-    onChange: {
-      name: 'onChange',
+    blrChange: {
+      name: 'blrChange',
```

Comment:
name overwrite the origin property name and show us in storybook what we want to see (instead of the origin).  I used it sometimes to show sizeVariant in the documentation correctly. In that case we don't need this line because the both have the same value."name"  is no longer needed and can be removed.

Reply:
Thanks for the hint, adjusted now for all the events.